lesson04/assignment/css/_settings.variables.scss

```diff
@@ -0,0 +1,16 @@
+// Colors
+$darkorange: #ff8c00;
+$ivory: #fffff0;
+$lightgray: #dcdcdc;
+$dimgray: #696969;
+$midnightblue: #191970;
+$aliceblue: #f0f8ff;
+$black: #000000;
+$white: #ffffff;
+
+
+$font-fam: "arial, helvetica, sans-serif";
+
+$a:focus: #696969
```

Comment:
I am not sure if this syntax works.  Don't think you can apply the focus to the variable itself.  There is also a missing semi colon at the end of the line.   I would do something like this:
`$link-color: #696969;
  $link-focus-color: #232323;
  a{
    color: $link-color;
  }
  a:focus{
    color: $link-focus-color;
  }
`